astropy/io/ascii/tests/test_read.py

```diff
@@ -223,7 +218,7 @@ def test_read_all_files(fast_reader, path_format, home_is_data):
             table = ascii.read(testfile["name"], **test_opts)
             assert_equal(table.dtype.names, testfile["cols"])
```

Comment:
```suggestion
            assert table.dtype.names == testfile["cols"]
```
These are tuples of strings, so better compared with the standard `assert`